suites/e2e/tests/serial/index.js

```diff
@@ -16,31 +16,21 @@
 
 const { delay } = require('bluebird');
 const util = require('util');
-const exec = util.promisify(require('child_process').exec);
+const fs = require('fs').promises;
 
 module.exports = {
 	title: 'Serial Test',
 	tests: [
 		{
 			title: 'Recording DUT serial output',
-			run: async function(test) {
+			run: async function (test) {
 				await this.context.get().worker.on();
-				await delay(20 * 1000);
-				await exec(
-					'head -n 20 /reports/dut-serial.txt',
-					(error, stdout, stderr) => {
-						if (error || stderr || stdout === '') {
-							throw new Error(
-								`Error with DUT serial output: ${error + stderr + stdout}`,
-							);
-						}
-						// console.log(stdout)
-						test.true(
-							stdout,
-							'Should be able to record serial output from DUT.',
-						);
-						console.log(done);
-					},
+				await delay(10 * 1000);
+
+				test.not(
+					(await fs.stat('/reports/dut-serial.txt')).size,
```

Comment:
Very good choice @Bucknalla This covers every scenario I have in mind. Did you test false positives with this? Is the test adequate?

Example: Run the test but pull the serial cable out. Does it fail properly.

Reply:
If you pull the cable out during the test it won't fail if some of the logging was accomplished but we can't really handle for changing circumstances during a test. This has been tested with and without the cable being attached so it's a good fire test to see if logging is working. I mean we could change it to use a threshold instead of like minimum filesize, that would account for any noise or interference picked up but that's pretty unlikely.

Reply:
Didn't mean that we should be pulling the cable out in the middle of the test. Your scenario is what I meant. 
And, yeah I don't think we can account for or test for interference/noise.

Reply:
Resolving